pulp_rpm/app/migrations/0007_advisory_pkg_sumtype_as_int.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.11 on 2020-04-06 16:05
+
+from django.db import migrations, models, transaction
+
+
+def translate_sum_type(apps, schema_editor):
+    # use sum_type as int as createrepo_c uses
+    with transaction.atomic():
+        UpdateCollectionPackage = apps.get_model('rpm', 'UpdateCollectionPackage')
+        for package in UpdateCollectionPackage.objects.all():
+            package.sum_type = int(package.sum_type) if package.sum_type else 0
```

Comment:
I think no sum_type is sum_type=None and sum_type=0 means that we have a checksum, just of unknown type.